src/printer.js

```diff
@@ -76,7 +77,29 @@ function genericPrint(path, options, print) {
       comments.printDanglingComments(path, options, /* sameIndent */ true)
     ]);
   }
-  const printed = printNode(path, options, print);
+  // if this is embedded code, lets try to match the indentation of the open <?php tag
+  let printed;
+  if (isPrevNodeInline(path)) {
+    const tagOpenIndex = options.originalText.lastIndexOf(
+      "<",
```

Comment:
Let's add test: `<?php echo '<'; ?><?php echo '<'; ?>`. I don't sure what using only `<` is good solution